src/Symfony/Component/Form/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -80,7 +86,13 @@ public function reverseTransform($value)
         $value = parent::reverseTransform($value);
 
         if (null !== $value) {
```

Comment:
I'd propose to change this to `if (null !== $value && 1 !== $this-divisor)` as the whole divisor logic does not make sense when it's 1 (the default value). please do the same in the `transform` method as well.